app/views/dashboard/index.html.erb

```diff
@@ -11,14 +11,22 @@
         <p><%= party.party_date.strftime("%B %e, %Y") %></p>
         <p><%= party.start_time %></p>
         <p><%= current_user.party_status(party.id) %></p>
+        <% if party.any_guests? %>
+          <h6>Guests:</h6>
+          <ul>
+            <% party.invitees.each do |invitee| %>
+              <li><%= invitee.email %></li>
+            <% end %>
+          </ul>
+        <% end %>
```

Comment:
Added functionality that all guests will display on a party detail element (not part of user story)